templates/helper_spec_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module HelperSpecHelpers
+  def stub_current_user(as: nil)
```

Comment:
A good number, I hope!

Clearance was the most obvious place to me, but I'd like to find homes for methods like this (even though they're small) in other projects. It's really easy for Suspenders to become bloated, and I don't want our new Rails apps to start out with a hundred lines of actual code we need to maintain. People complain enough already about the number of gems we include.

If we want a Clearance-agnostic approach for this helper, maybe something like Shoulda makes sense for this?